packages/playwright/src/common/testType.ts

```diff
@@ -239,7 +239,7 @@ export class TestTypeImpl {
       throw new Error(`test.${type}() can only be called inside test, describe block or fixture`);
     if (typeof modifierArgs[0] === 'function')
       throw new Error(`test.${type}() with a function can only be called inside describe block`);
-    testInfo._modifier(type, location, modifierArgs as [any, any]);
+    testInfo[type](...modifierArgs as [any, any]);
```

Comment:
To be honest, I liked the `testInfo._modifier` because it's proper, but oh well 😄